src/box/schema_def.h

```diff
@@ -57,7 +57,13 @@ enum {
 	/** Yet another arbitrary limit which simply needs to
 	 * exist.
 	 */
-	BOX_INDEX_PART_MAX = UINT8_MAX
+	BOX_INDEX_PART_MAX = UINT8_MAX,
+	/**
+	 * Start of the range of space ids reserved for special spaces whose
+	 * metadata is not replicated. These spaces need a special range of ids
+	 * to avoid conflicts with spaces which could arrive via replication.
+	 */
+	BOX_SPACE_ID_META_TEMPORARY_MIN = (1 << 30),
```

Comment:
> Нужен нормальный флаг для таких спейсов, в их опциях, и больше ничего. Не надо айди никак фиксировать.

Are you sure it is done? Or did I misread the comment? You are saying: "ids reserved for special spaces". But it not reserved. It is just advisory range used by default. Users still can choose their own IDs, can they? (Sorry, I didn't review the whole patch, just looking at certain pieces of it.)

Reply:
Ids are no longer required to be in this range, so maybe the comment needs a bit more rewording. But having a default range and it being defined somewhere in tarantool (not just lua module) seems to be a good idea

Reply:
I updated the comment, is there something else to this discussion?

Reply:
I will check during the next review and resolve all the closed comments myself. Have very little time for reviews ATM, unfortunately.